deploy/helm/lunettes/templates/lunettes/lunettes-deploy.yaml

```diff
@@ -78,7 +78,7 @@ spec:
         - --trace-enable={{ .Values.traceEnable }}
         - --pod_info_cache_size=100000
         - --doc_num_peer_query=200
-        - --num_scroll_slice=8
```

Comment:
can you please add documentation and explain about this parameter change, thanks